Makefile

```diff
@@ -31,3 +31,8 @@ server:
 .PHONY: clean
 clean:
 	yarn run clean
+
+.PHONY: test_unit_codecov
+test_unit_codecov: test_unit
+	npm install codecov -g
```

Comment:
This should just be added to the project dependencies, not installed every time we run make. Also, we use `yarn` for dependency management. Using `npm` here will probably generate a separate lock file that is incompatible with our yarn lockfile.